types/2019-12-03/InvoiceItems.d.ts

```diff
@@ -159,6 +154,11 @@ declare module 'stripe' {
        */
       amount?: number;
 
+      /**
+       * Three-letter [ISO currency code](https://www.iso.org/iso-4217-currency-codes.html), in lowercase. Must be a [supported currency](https://stripe.com/docs/currencies).
+       */
+      currency?: string;
```

Comment:
this didn't have an analog in the stripe-java PR? Is it related to the missing events?

Reply:
Nah, it's the ordering of the field that seems to have changed for reasons beyond me, but it doesn't look like anything bad right?

Reply:
It also is changing to optional